Azure.OpenAI/Client/Extensions/Localization.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) David Pine. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.OpenAI.Client.Extensions;
+
+internal sealed class Localization
+{
+
+    internal  static async Task ConfigCulture(WebAssemblyHost host)
```

Comment:
```suggestion
    internal static async Task ConfigCulture(this WebAssemblyHost host)
```

This is in the _Extensions_ folder, but it's not an extension. Let's change that.